### ISSUE TITLE
Restore master branch for backwards compatibility

### ISSUE THREAD

**Issue by alanj853** [nerves-project/shoehorn#37]
Hi,

Could the master branch please be restored for backwards compatibility?

I have an application which uses the master branch. While I can switch it over to using "main" from now on, previous releases I have will no longer work as master no longer seems to exist.

**Comment by fhunleth** [comment]
That wouldn't work anyway. The master branch doesn't point to one commit. It changed every time a commit was made, so even if we were to restore it, it would be to a different commit than whatever it pointed to the day that the release was made.

**Comment by alanj853** [comment]
So this is a strange situation that actually seemed to resolve itself for us.

In our mix.exs file, we had shoehorn set to pull from this repository, with no branch specified, so it was pulling from master (at commit 77dd0bfe4abdf88a2773220ecb78ed0f0dd107fa). So our mix.lock also had no branch specified, and just pointed to a commit.

When I tried building yesterday, my 'mix deps.get' failed because there was no remote "master" branch anymore. When I tried to build today, it worked fine with your "main" branch (because I see the commit still exists in the repo, on the "main" branch now). So, my problem is now solved.

But on the backwards compatibility issue, I know the master branch doesn't point to one commit. But lets say I had decided to specify "master" as the branch to check out in the mix.exs file in previous releases, then I would have a problem because none of my previous releases would build now, without having to re-release/hotfix to use your new branch.

As I said, my team and I are okay now and don't need this issue solved for us, but just said I'd let you know, as it might affect others. Obviously, it is your choice though.
